README.md

```diff
@@ -175,6 +179,21 @@ $> inertia remote status local
 Remote instance 'local' accepting requests at http://0.0.0.0:8081
 ```
 
+If you run into this error when deploying onto the `testvps`:
+
+```bash
+docker: Error response from daemon: error creating aufs mount to /var/lib/docker/aufs/mnt/fed036790dfcc73da5f7c74a7264e617a2889ccf06f61dc4d426cf606de2f374-init: invalid argument.
+```
+
+You probably need to go into your Docker settings and add this line to the Docker daemon configuration file:
```

Comment:
It wasn't immediately clear to me where this was. Docker went out of their way to make it a part of the Taskbar tray, they must really want you to edit it there. Maybe we should leave the directions here a bit more explicitly.

Docker -> Preferences -> Daemon -> Advanced -> Edit File.